grafana_collector/grafana/time.go

```diff
@@ -67,6 +67,11 @@ const (
 	boundaryTimeRegExp = "^(.*?)/([dwMy])$"
 )
 
+// NewIteration ... returns the number of milliseconds elapsed since January 1, 1970 UTC.
+func NewIteration() int {
+	return int(time.Now().UnixNano() / int64(time.Millisecond))
```

Comment:
Why not use `time.Now().Second()`?

Reply:
Second returns the second offset，in the range [0, 59]. But I need to the number of milliseconds elapsed since January 1, 1970 UTC.